package.json

```diff
@@ -70,6 +71,9 @@
     "bcrypt-nodejs": "0.0.3",
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.18.2",
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
+    "chance": "^1.1.4",
```

Comment:
Two things: first remove unused dependencies (like chai.http) and also some of this should be devDependencies instead